WordPress/src/main/java/org/wordpress/android/ui/history/HistoryDetailActivity.kt

```diff
@@ -19,6 +20,9 @@ class HistoryDetailActivity : LocaleAwareActivity() {
             setContentView(root)
             setSupportActionBar(toolbarMain)
         }
+
+        onBackPressedDispatcher.addCallback(this) { AnalyticsTracker.track(Stat.REVISIONS_DETAIL_CANCELLED) }
```

Comment:
Warning (⚠️): As per the other such changes you did to replicate the `super.onBackPressed()` behavior, it seems that the `isEnabled = false` + `onBackPressedDispatcher.onBackPressed()` additions are missing from this callback.

PS: Same applies on the `SuggestionActivity.kt` class and its `onBackPressedDispatcher.addCallback(this) { viewModel.trackExit(false) }` replacement, which too seems to be missing the above.

Reply:
Good catch! I added the missing ones: cc9f2e9b0570f21c1bb4d4dd6ba2b159a8cdcb31

Reply:
Awesome, thanks for the fix @irfano ! 🥇